src/web/components/ApiKeyManagement/KeyHelper.ts

```diff
@@ -1,12 +1,12 @@
 import { ApiKeyDTO } from '../../../api/services/adminServiceHelpers';
 
-const yearInSeconds = 31556952;
+export const yearInSeconds = 60 * 60 * 24 * 365.25;
 
-export const needsRotating = (apiKey: ApiKeyDTO): boolean => {
-  const currentDate = new Date().getTime();
+// api key is recommended to rotate if older than one year
```

Comment:
I think we can get rid of this comment - the tests should describe the behaviour clearly enough. Otherwise if we change the expiry time, then we'd need to update this comment as well.